tools/update-dll-msvc.bat

```diff
@@ -0,0 +1,176 @@
+@echo off
+rem Update the DLL using MSVC.
+rem current support version of Visual C compiler is 2008 and upper version
+rem
+rem usage: update-dll-msvc
+rem
+rem
+rem Sample .vimrc:
+rem
+rem NeoBundle 'Shougo/vimproc.vim', {
+rem \ 'build' : {
+rem \     'windows' : 'tools\\update-dll-msvc',
+rem \     'cygwin' : 'make -f make_cygwin.mak',
+rem \     'mac' : 'make -f make_mac.mak',
+rem \     'linux' : 'make',
+rem \     'unix' : 'gmake',
+rem \    },
+rem \ }
+
+REM --------------------------------------------------------------------------------------------
+REM Set the default value for vimproc_arch and msvc_arch based on processor architecture.
+REM --------------------------------------------------------------------------------------------
+set vimproc_arch=32
+set msvc_arch=x86
+if /i "%PROCESSOR_ARCHITECTURE%"=="AMD64" set "vimproc_arch=64" & set "msvc_arch=x86_amd64" & goto Search_VC_Location
+if /i "%PROCESSOR_ARCHITEW6432%"=="AMD64" set "vimproc_arch=64" & set "msvc_arch=x86_amd64" & goto Search_VC_Location
+if /i "%PROCESSOR_ARCHITECTURE%"=="x64"   set "vimproc_arch=64" & set "msvc_arch=x86_amd64" & goto Search_VC_Location
+if /i "%PROCESSOR_ARCHITECTURE%"=="IA64"  set "vimproc_arch=64" & set "msvc_arch=x86_amd64" & goto Search_VC_Location
+if /i "%PROCESSOR_ARCHITEW6432%"=="IA64"  set "vimproc_arch=64" & set "msvc_arch=x86_amd64"
```

Comment:
I don't think we need to support IA64.
If such support is needed, msvc_arch should be set to "ia64".
However, recent versions of VC don't support targeting IA64.

Reply:
@k-takata , OK, I will remove the support of IA64 as your suggestion.

Reply:
@k-takata , I have just check with Windows SDK 7.0 with Visual C compiler version is 9.0, it support IA64. I will recommend to keep it and change it from x86_amd64 to x86_ia64.

Reply:
We haven't tested vimproc on IA64 at all, and I don't want to support it.